Rakefile

```diff
@@ -49,6 +42,7 @@ namespace :import do
         to_index.push(row["nct_id"])
         tally = -> {}
         begin
+          user = User.find_by(email: row["user_email"])
```

Comment:
This will hit the database for the user once for each row and it would be faster to cache the users in a map after you see them.

Reply:
yeah, agree but that exceeds my rails skills and I figured we are only doing a few thousand at a time, so it wastes a few electrons but no big deal. If we are going to get crazy, we could just require the id instead of email and insert directly to the db.